.github/workflows/nightly-flake.yaml

```diff
@@ -0,0 +1,37 @@
+name: nightly-flake
+on:
+  schedule:
+    # Every day at midnight
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+  push:
+    paths:
+      # For testing purposes
+      - ".github/workflows/nightly-flake.yaml"
+jobs:
+  test-go-race:
+    runs-on: 'buildjet-8vcpu-ubuntu-2204'
```

Comment:
Are the high cpu GHA runners flakier than the buildjet ones?

Reply:
Yeah, but I wanted to keep things the same to make so that only real threats are shown. I'll add a comment.